[AssetMapper] Put importmap in polyfill so it can be hosted locally easily (#19057)
symfony/symfony-docs: frontend/asset_mapper.rst

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Configure the polyfill for older browsers. Default is `ES module shim`_. You can pass

Reviewer: ```suggestion
Configure the polyfill for older browsers. By default, the `ES module shim`_ is loaded via a CDN. You can pass
```

> an importmap name to load the polyfill, or ``false`` to disable the polyfill loading.

Reviewer: ```suggestion
the key of an item in ``importmap.php`` or ``false`` to disable the polyfill loading.
```
